### ISSUE TITLE
Expose payment requirement to listing page

### ISSUE THREAD

**Issue by wagnerand** [mozilla/addons-server#4955]
Following #4954, we need to expose the payment requirement flag on the public listing page.

Our official policy would be like:
A payment is required to use this add-on or the service this add-on relies on, either upfront, after a trial period or in conjunction with a paid software.

This sounds a bit too wordy to me. I think we could simply and go by:
A payment is required to use this add-on or the service it relies on.

//cc @jvillalobos 
//cc @pwalm for UX (on the new desktop pages)
Should we expose it on mobile as well?

**Comment by wagnerand** [comment]
Ugh, this should probably be on the frontend..

**Comment by eviljeff** [comment]
fwiw, addons-frontend is only used for mobile pages at the moment - if this needs to be exposed to Desktop users (before the eventual rewrite at some point) then it needs implementing on addons-server too.

**Comment by wagnerand** [comment]
Yeah, I was assuming that we'll have the new desktop design before we have post-review. Am I wrong?

**Comment by eviljeff** [comment]
I don't know about the relative priority of either project.  Can you edit the issues to mention this is explicitly connected to post-review also

**Comment by jvillalobos** [comment]
> Yeah, I was assuming that we'll have the new desktop design before we have post-review. Am I wrong?

Yes, launching the new desktop site to the public is unlikely to happen in Q2.

**Comment by wagnerand** [comment]
After giving it some more thought, I don't think we have to tie this to post-review.
